ariadne/asgi.py

```diff
@@ -89,6 +89,7 @@ def __init__(
         extensions: Optional[Extensions] = None,
         middleware: Optional[Middlewares] = None,
         keepalive: float = None,
+        **graphql_kwargs: Any,
```

Comment:
This will make any typos in options be eaten up and passed all the way down to `graphql()` call. We should be explicit in what options we take.